trl/trainer/grpo_trainer.py

```diff
@@ -101,6 +101,9 @@ class GRPOTrainer(Trainer):
                   [Using a custom reward function](#using-a-custom-reward-function).
             - A list of reward functions, where each item can independently be any of the above types. Mixing different
             types within the list (e.g., a string model ID and a custom reward function) is allowed.
+        reward_weights (`list[Union[float, int]]`, *optional*, defaults to `None`):
```

Comment:
I'd go for float only

```suggestion
        reward_weights (`list[Union[float]]`, *optional*, defaults to `None`):
```